packages/rollup-contracts/contracts/RollupMessagePasser.sol

```diff
@@ -0,0 +1,22 @@
+pragma solidity ^0.5.0;
+pragma experimental ABIEncoderV2;
+
+/* Internal Imports */
+import {DataTypes as dt} from "./DataTypes.sol";
+
+contract RollupMessagePasser {
```

Comment:
This could probably be named a bit better. I really like `L1ToL2Message` because it indicates exactly how it is to be used. The same can be done for this class. If the intention is to ultimately invoke L2 functions, we could probably be more specific than `message`. Maybe something like `OvmTxBridge` or `L2TransactionExecutor` would be clearer? 🤷‍♂

Reply:
I also like `L1ToL2MessagePasser`. I think it's pretty conceptually easy to think of L1 and L2 as some abstract "parent" and "child" relationship between blockchains so maybe good to make use of that intuition.

Also, it is kind of interesting to replace `message` with `transaction`, but then again idk if that gets confusing with the already overloaded term....... But I guess "cross chain transaction" is already a thing. We could totally call this `L1ToL2Transaction`....

Anyway, I'm open to options!